src/tests/roc_fec/target_openfec/test_writer_reader.cpp

```diff
@@ -941,11 +983,14 @@ TEST(writer_reader, resize_source_block_begin) {
             CHECK(p);
             CHECK(p->fec());
             CHECK(p->fec()->source_block_length == block_sizes[n]);
+
+            check_audio_packet(p, rd_sn);
+            rd_sn++;
```

Comment:
It is worth to check if `wr_sn == rd_sn` at the end.

Reply:
Indeed, done.